src/mozanalysis/metrics/desktop.py

```diff
@@ -61,7 +65,7 @@
                     *,
                     DATE(submission_timestamp) AS submission_date,
                     environment.experiments
-                FROM `moz-fx-data-shared-prod`.telemetry_stable.main_v4
+                FROM mozdata.telemetry_stable.main_v4
```

Comment:
This needs to be `moz-fx-data-shared-prod`. `mozdata` doesn't have access to stable or derived tables

```suggestion
                FROM `moz-fx-data-shared-prod`.telemetry_stable.main_v4
```

Reply:
thanks for the catch!